pkg/handler/login.html

```diff
@@ -1,19 +1,60 @@
 <!doctype html>
-<html lang="en">
+ <html lang="en">
 
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Github Example</title>
-    <style>
-        a.button {
-            text-decoration: none;
-        }
-    </style>
-</head>
+ <head>
+     <meta charset="utf-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <title>Login with Github</title>
+     <style>
+         html, body {
+             background-color: #efefef;
+             display: flex;
+             flex-direction: column;
+             font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+             height: 100%;
+             justify-content: center;
+         }
 
-<body>
-    <a href="/github/login" class="button">Login with Github</a>
-</body>
+         .title {
+             font-weight: 400;
+             margin: 20px auto;
+         }
 
-</html>
\ No newline at end of file
+         .container {
+             background-color: #fff;
+             border: 1px solid #ddd;
+             border-radius: 3px;
+             display: flex;
+             flex-direction: column;
+             justify-content: center;
+             margin: 0 auto;
+             padding: 20px;
+         }
+
+         .message {
+             margin-bottom: 30px;
+         }
+
+         .button {
+             background-color: #2ebc4f;
+             border-radius: 3px;
+             color: #fff;
+             display: inline-block;
+             margin: 0 auto;
+             padding: 20px 32px;
+             text-decoration: none;
+         }
+     </style>
+ </head>
+
+ <body>
+     <h1 class="title center">Login to Tail</h1>
+     <div class="container center">
+         <p class="message center">
+             Please note that only Kubermatic organization members are allowed to access Tail.
```

Comment:
Perhaps you can come up with a better idea.

Reply:
Maybe we can make the file a template and have the org name be a template variable?

Reply:
> Maybe we can make the file a template and have the org name be a template variable?

Yeah, this would be cool. We are trying to make it as much customizable as possible, in case anyone else decides to use it. :)

Reply:
+1 from me, should I do it or will you take care of it later? I do not have setup and this file was created in GitHub editor so perhaps you can do it a bit quicker 🙂

Reply:
> will you take care of it later? 

I'll do it. :)

Reply:
Great, thanks :)